tap_dbt/streams.py

```diff
@@ -69,11 +73,15 @@ def get_url_params(
         """Return offset as the next page token."""
         params = {}
         _ = context
+        params["limit"] = self.config.get("page_size", 100)
```

Comment:
The config json schema's `default` will ensure this is always set:

```suggestion
        # TODO: Get page size from the pagination object when it's available in this scope (https://github.com/meltano/sdk/issues/1606)
        params["limit"] = self.config["page_size"]
```